.gitignore

```diff
@@ -25,3 +25,4 @@ build/Release
 node_modules
 public/bower_components
 reports
+programmer.css
```

Comment:
Something generates `programmer.css` at the root??

Reply:
No, this file is generated in public/stylesheets/programmer.css

This line in the .gitignore ignores all programmer.css in any place.

Do you think that we should be more specific ?